packages/web/src/pages/remixes-page/hooks.ts

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'react'
+
+import { useSearchParams as useParams } from 'react-router-dom-v5-compat'
+
+export type RemixSortMethod = 'likes' | 'plays' | 'recent'
+
+export const useRemixPageParams = () => {
+  const [urlRemixPageParams] = useParams()
+
+  const sortMethod = urlRemixPageParams.get('sortMethod') as RemixSortMethod
+  const isCosign = urlRemixPageParams.get('isCosign')
+  const isContestEntry = urlRemixPageParams.get('isContestEntry')
+
+  const remixPageParams = useMemo(
+    () => ({
+      sortMethod: sortMethod || undefined,
+      isCosign: isCosign || undefined,
+      isContestEntry: isContestEntry || undefined
```

Comment:
these will return `undefined` on a `false` value - I assume this is probably what you want instead

```suggestion
      isCosign: isCosign ?? undefined,
      isContestEntry: isContestEntry ?? undefined
```

Reply:
oh yea i copied these from search but i'll clean this up upstream so these checks aren't necessary